site/content/en/docs/Reference/Drivers/kic.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "kic"
+linkTitle: "kic"
+weight: 3
+date: 2017-01-05
+date: 2019-11-06
+description: >
+  kubernetes in container. vm-free.
+---
+
+## Overview
+
+This is an experimental driver. Run kubernetes inside a container, instead of creating a VM, kubernertes will be installed in container.
```

Comment:
Check spelling of Kubernetes, and see if you can rephrase it in such a way that only says Kubernetes once in a sentence.